Assets/BossRoom/Scripts/Client/UI/PopupPanel.cs

```diff
@@ -355,6 +359,38 @@ void SetupEnterGameDisplayForUnityRelay()
             m_PortInputField.text = "";
         }
 
+        async Task<bool> UnityAuthenticationAnonymousSignIn()
+        {
+            try
+            {
+                if (UnityServices.State == ServicesInitializationState.Uninitialized)
+                {
+                    await UnityServices.InitializeAsync();
+                }
+
+                if (!AuthenticationService.Instance.IsSignedIn)
+                {
+                    await AuthenticationService.Instance.SignInAnonymouslyAsync();

```

Comment:
I would expect the Auth library to have gates like this to prevent accidental duplicate login.  maybe I'm wrong, but it's what I'd want to have. :)

Reply:
This class was deleted, brought back by git user error